pkg/pool/operations/import.go

```diff
@@ -76,8 +83,9 @@ func (oc *OperationsConfig) Import(cspi *cstor.CStorPoolInstance) (bool, error)
 		cmdOut, err = zfs.NewPoolImport().
 			WithDirectory(devID).
 			WithProperty("cachefile", cacheFile).
-			WithPool(PoolName()).
-			WithExecutor(oc.zcmdExecutor).
+			WithPool(existingPoolName).
```

Comment:
Since it was scanning through all the blockdevices(If bad disk exists then migration will not be successful). Good to have a **note** saying that before performing migration make sure that there shouldn't be any bad disks attached to the node.
To make sure they can run `sudo fdisk -l` on all the CStor-pool pod nodes if command hung then there will is some problem.
cc: @sonasingh46

Reply:
Sure will add this to migration documentation.